src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -202,19 +232,21 @@ different reconstruction algorithms (for example ones using motion correction).
 
 Currently supported image contrasts include:
 
-| **Name**  | `contrast_label` | **Description**                                                                                      |
-|---------- | ---------------- | ---------------------------------------------------------------------------------------------------- |
-| BOLD      | bold             | Blood-Oxygen-Level Dependent contrast (specialized T2\* weighting)                                   |
-| CBV       | cbv              | Cerebral Blood Volume contrast (specialized T2\* weighting or difference between T1 weighted images) |
-| Phase     | phase            | Phase information associated with magnitude information stored in BOLD contrast                      |
+| **Name**  | `contrast_label` | **Description**                                                                                                                                                                                                     |
+|---------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| BOLD      | bold             | Blood-Oxygen-Level Dependent contrast (specialized T2\* weighting)                                                                                                                                                  |
+| CBV       | cbv              | Cerebral Blood Volume contrast (specialized T2\* weighting or difference between T1 weighted images)                                                                                                                |
+| Phase     | phase            | Phase information associated with magnitude information stored in BOLD contrast. [Deprecated](../02-common-principles.md#definitions). [`part-phase`](../99-appendices/09-entities.md#part) should be used instead. |
```

Comment:
I think we can be a little BOLDER about this, and more explicit about using the `bold` suffix.

```suggestion
| Phase     | phase            | [DEPRECATED](../02-common-principles.md#definitions). Phase information associated with magnitude information stored in BOLD contrast. This suffix should be replaced by the [`part-phase`](../99-appendices/09-entities.md#part) in conjunction with the `bold` suffix. |
```